### DISCUSSION TITLE
0.14 Dark Mode Text Legibility

### DISCUSSION THREAD

**Discussion by maxwood** [blakeblackshear/frigate#13410] [Ideas]
0.14 is brilliant but I just have a few small ideas that would make a big difference - particularly in dark mode. 

The dark grey text colour used throughout is very hard to read, would it be possible to get this a shade brighter?

Would it be possible to widen the colour/highlight section on the timeline (narrow strip far right) it's quite hard to see clearly. (This would also apply to light mode)

Lastly, the time stamps on the timeline are also a very small font and the text colour very dark.

Thanks so much for your efforts and hard work!

![image](https://github.com/user-attachments/assets/7efb3173-3edb-4371-ba59-dd66e391c6fc)

**Comment by hawkeye217** [comment]
There is a High Contrast theme under the themes menu that increases readability. Give it a try.

**Reply by maxwood** [reply]
> There is a High Contrast theme under the themes menu that increases readability. Give it a try.

Ahhhh I was just coming here to post that I literally just found this seconds ago! Thanks for the heads up :)


For anyone else.. 

![image](https://github.com/user-attachments/assets/eb4097f4-f5e6-4a03-8a54-fe0dcee709e5)

**Comment by NickM-27** [comment]
There is a high contrast theme available, in general things are designed and implemented as intended
